src/main/interface/settings.c

```diff
@@ -447,7 +447,8 @@ const clivalue_t valueTable[] = {
     { "gyro_lma_depth",             VAR_UINT8  | MASTER_VALUE, .config.minmax = {0, 11}, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_lma_depth)},
     { "gyro_lma_weight",            VAR_UINT8  | MASTER_VALUE, .config.minmax = {0, 100}, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyro_lma_weight)},
 
-    { "moron_threshold",            VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0,  200 }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyroMovementCalibrationThreshold) },
+    { "gyro_calib_duration",        VAR_UINT16 | MASTER_VALUE, .config.minmax = { 50,  3000 }, PG_GYRO_CONFIG, offsetof(gyroConfig_t, gyroCalibrationDuration) },
```

Comment:
30 sec max? That's a lot, don't think it's useful.

Reply:
Have to use a uint16 anyway to support the expressed desire to be able to use 10 seconds in some use cases.  So making it available up to 30 seconds doesn't really hurt anything - nobody is forced to use it.  I originally just went for the max int16 but decided that 327 seconds was probably overkill. ;)